tensorrt_llm/_torch/pyexecutor/hang_detector.py

```
@@ -113,11 +114,22 @@ def run_loop():
     async def _detect_hang(self):
         await asyncio.sleep(self.timeout)
         with self.lock:
             self._detected = True
             logger.error(f"Hang detected after {self.timeout} seconds.")
+            for provider in self._status_providers:
+                try:
+                    status = provider()
+                    if status:
+                        logger.error(status)
+                except Exception:
+                    pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)